.github/ISSUE_TEMPLATE/data_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug - data
+about: Report a bug with your data build
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+**Please complete the following information**
+Data repository URL: [e.g. github.com/ONSdigital/sdg-indicators]
+Data or metadata issue:
+Input format: [e.g. CSV, YAML (Markdown), SDMX]
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
I think maybe we could remove "To Reproduce", "Expected behavior", and "Screenshots". These seem more relevant for site bugs. What do you think?

Reply:
Think that makes sense!